cmd/apprepository-controller/pkg/client/clientset/versioned/typed/apprepository/v1alpha1/apprepository_client.go

```diff
@@ -71,7 +70,7 @@ func setConfigDefaults(config *rest.Config) error {
 	gv := v1alpha1.SchemeGroupVersion
 	config.GroupVersion = &gv
 	config.APIPath = "/apis"
-	config.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: scheme.Codecs}
+	config.NegotiatedSerializer = scheme.Codecs.WithoutConversion()
```

Comment:
Most of the modifications in [that commit](https://github.com/kubeapps/kubeapps/pull/1359/commits/1e1a3ba7c4c3c7b0a177adcffec07169847c1a84) were suggested by @andresmgot so that we could import `helm.sh/helm/v3` at all.